packages/__vital__/src/shadow/@bodiless/vital-meta/MetaHelmet.ts

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright © 2022 Johnson & Johnson
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { withMeta } from '@bodiless/components';
+import { asElementToken } from '@bodiless/vital-elements';
+import {
+  vitalMetaHelmetBase,
+  WithHtml,
+  WithHomePageSchemas,
+  WithPageDescription,
+  WithPageTitle,
+  WithSeoForm,
+  WithShareForm,
+  WithShareDescription,
+  WithShareImage,
+  WithShareTitle,
+  WithShareType,
+  WithShareUrl,
+  WithSiteName,
+  WithTwitterCard,
+  WithTwitterTitle,
+  WithUTMCampaign,
+  WithUTMContent
+} from '@bodiless/vital-meta';
+import { Token, as } from '@bodiless/fclasses';
+
+const asSimpleToken = (...tokens: Token[]) => asElementToken({
+  Core: {
+    _: Array.isArray(tokens) ? as(...tokens) : tokens,
+  },
+});
+
+const WithMetaKeywords = asSimpleToken(withMeta({
+  name: 'keywords',
+  label: 'Keywords',
+  placeholder: 'No more than 10 keyword phrases',
+})('page-keywords'));
+
+/*
```

Comment:
@wodenx  This is how I thought it should work and finally got it to work by recomposing the whole thing.